_test/common_functions/clean_up_file.m

```diff
@@ -9,7 +9,11 @@ function clean_up_file(file_path)
 end
 
 % Get list of all open file IDs and the names of the files
-all_open_fids = fopen('all');
+if verLessThan('MATLAB','24')
```

Comment:
Required change: Make the Matlab version explicit. R2024a is 24.1, R2024b is 24.2. Do you mean 24.1 or 24.2? It is confusing because '24' is not a formal Matlab version (see the Table in https://en.wikipedia.org/wiki/MATLAB). In my opinion, to make the condition readable, it is much better to use 'R2024a' or 'R2024b', except possibly where code is performance critical, which is not the case here.